pom.xml

```diff
@@ -6,7 +6,7 @@
   <parent>
     <groupId>org.springframework.boot</groupId>
     <artifactId>spring-boot-starter-parent</artifactId>
-    <version>3.0.2</version>
+    <version>3.0.3</version>
```

Comment:
latest version is 3.0.4. we could also keep it at 3.0.2 and bump dependencies to 3.0.4 in all tools at the same time.